exercises/practice/bob/.meta/example.sql

```diff
@@ -0,0 +1,27 @@
+WITH trimmed AS (
+    SELECT input
+         , RTRIM(input, CHAR(32, 9, 10, 13)) AS trimmed
+    FROM heyBob
+)
+, bools AS (
+    SELECT input
+         , trimmed = '' AS is_silence
+         , trimmed LIKE '%?' AS is_question
+         , trimmed GLOB '*[a-zA-Z]*' AND trimmed = UPPER(trimmed) AS is_yelling
+    FROM trimmed
+)
+UPDATE heyBob
```

Comment:
Are we going with lower camelCase for table naming in this track? It looks like you're using `snake_case` for columns.

Reply:
That's a good point. IMO snake_case is the way to go. We don't want case sensitive table or column names.